Update Flink integration docs to 1.16 (#13501)
DataDog/integrations-core: flink/README.md

_Available for Agent >6.0_

> 1. Flink uses the `log4j` logger by default. To activate logging to a file and customize the format edit the `log4j*.properties` configuration files in the `conf/` directory of the Flink distribution. See the [Flink logging documentation][13] for which configuration file is relevant for your setup. See [Flink's repository][6] for default configurations.

Reviewer: ```suggestion
1. Flink uses the `log4j` logger by default. To enable logging to a file, customize the format by editing the `log4j*.properties` configuration files in the `conf/` directory of the Flink distribution. See the [Flink logging documentation][13] for information on which configuration file is relevant for your setup. See [Flink's repository][6] for default configurations.
```

Reviewer: I could be very wrong with my interpretation of the above step. Please let me know if that's the case.

1. Flink uses the `log4j` logger by default. To activate logging to a file and customize the format edit the `log4j.properties`, `log4j-cli.properties`, `log4j-yarn-session.properties`, or `log4j-console.properties` file. See [Flink's repository][6] for default configurations. For example `log4j.properties` contains this configuration by default:

   ```conf

>    appender.main.name = MainAppender

Author: Overall I'm not sure how much value this excerpt really adds, and it easily becomes out-dated.

Reviewer 2: @zentol sorry about the delay. Do you think it would be possible to instead link to some examples from Flink docs?

Author: I've updated the logging section. :+1:

Reviewer 2: Cool! I'll merge once the checks complete! Thanks again!
